demisto_sdk/commands/error_code_info/error_code_info.py

```diff
@@ -0,0 +1,62 @@
+from colorama import Fore
+from inspect import signature
+
+from demisto_sdk.commands.common import errors
+
+TEMPLATE = '''
+Error Code: {code}
+Function: {func}
+Message:
+{msg}
+'''
+
+
+def print_error_information(func_name, error_data, func, sig):
+    print(f'{Fore.GREEN}## Error Code Info ##{Fore.WHITE}')
+    print(TEMPLATE.format(
+        code=error_data['code'],
+        func=f'{func_name}{sig}',
+        msg=func(**{a: f'<{a}>' for a in list(sig.parameters)}),
+    ))
+
+
+def find_error(error_code):
+    for func_name, error_data in errors.ERROR_CODE.items():
+        if error_data['code'] == error_code:
+            return func_name, error_data
+
+    return '', {}
```

Comment:
you can use `get_error_object` found in errors.py

Reply:
thought about it but it doesn't return the function name.
I didn't want to change the signature of it.